source/features/quick-repo-deletion.tsx

```diff
@@ -64,7 +65,10 @@ async function buttonTimeout(buttonContainer: HTMLDetailsElement): Promise<boole
 		buttonContainer.open = false;
 	}, {once: true});
 
-	void verifyScopesWhileWaiting(abortController);
+	if(!await verifyScopes()) {
+		document.body.click()
```

Comment:
This trips `The modal was closed too quickly. Disabling close-out-of-view-modals for this session.` and I am not sure how to fix it

Reply:
I haven't look into the cause but this looks like a hack more than a fix.

Reply:
The overlay is because the button is in a "open" state. The fix should just ensure that the dropdown is closed when the promise is aborted.